crates/rpc/rpc/src/otterscan.rs

```diff
@@ -305,11 +307,141 @@ where
     /// Handler for `ots_searchTransactionsAfter`
     async fn search_transactions_after(
         &self,
-        _address: Address,
-        _block_number: u64,
-        _page_size: usize,
-    ) -> RpcResult<TransactionsWithReceipts> {
-        Err(internal_rpc_err("unimplemented"))
+        address: Address,
+        block_number: u64,
+        page_size: usize,
+    ) -> RpcResult<TransactionsWithReceipts<RpcTransaction<Eth::NetworkTypes>>> {
+        let state = self.eth.latest_state().map_err(|e| internal_rpc_err(e.to_string()))?;
+        let account = state.basic_account(&address).map_err(|e| internal_rpc_err(e.to_string()))?;
+
+        if account.is_none() {
+            return Err(EthApiError::InvalidParams(
+                "invalid parameter: address does not exist".to_string(),
+            )
+            .into());
+        }
+
+        let tip: u64 = self.eth.block_number()?.saturating_to();
+
+        if block_number > tip {
+            return Err(EthApiError::InvalidParams(
+                "invalid parameter: block number is larger than the chain tip".to_string(),
+            )
+            .into());
+        }
+
+        let mut txs_with_receipts = TransactionsWithReceipts {
+            txs: Vec::default(),
+            receipts: Vec::default(),
+            first_page: false,
```

Comment:
unclear what first_page means

Reply:
First page means it's the page with the most recent transactions meaning we have traced until the tip of the chain.